src/platform_impl/linux/drm/window.rs

```diff
@@ -0,0 +1,354 @@
+    pub fn id(&self) -> super::WindowId {
+        super::WindowId
+    }
+
+    #[inline]
+    pub fn set_title(&self, _title: &str) {}
+
+    #[inline]
+    pub fn set_visible(&self, _visible: bool) {}
+
+    #[inline]
+    pub fn is_visible(&self) -> Option<bool> {
+        Some(true)
+    }
+
+    #[inline]
+    pub fn outer_position(&self) -> Result<PhysicalPosition<i32>, NotSupportedError> {
+        Err(NotSupportedError::new())
+    }
+
+    #[inline]
+    pub fn inner_position(&self) -> Result<PhysicalPosition<i32>, NotSupportedError> {
+        Err(NotSupportedError::new())
+    }
+
+    #[inline]
+    pub fn set_outer_position(&self, _position: Position) {}
+
+    #[inline]
+    pub fn inner_size(&self) -> PhysicalSize<u32> {
+        let size = self.0.size();
+        PhysicalSize::new(size.0 as u32, size.1 as u32)
+    }
+
+    #[inline]
+    pub fn outer_size(&self) -> PhysicalSize<u32> {
+        self.inner_size()
+    }
+
+    #[inline]
+    pub fn set_inner_size(&self, _size: Size) {
+        // It's technically possible to do this by changing video modes but that seems a little
+        // restrictive
+    }
+
+    #[inline]
+    pub fn set_min_inner_size(&self, _dimensions: Option<Size>) {}
+
+    #[inline]
+    pub fn set_max_inner_size(&self, _dimensions: Option<Size>) {}
+
+    #[inline]
+    pub fn set_resizable(&self, _resizable: bool) {}
+
+    #[inline]
+    pub fn is_resizable(&self) -> bool {
+        false
+    }
+
+    #[inline]
+    pub fn set_cursor_icon(&self, _cursor: CursorIcon) {}
+
+    #[inline]
+    pub fn set_cursor_grab(&self, _grab: bool) -> Result<(), ExternalError> {
+        Err(ExternalError::NotSupported(NotSupportedError::new()))
+    }
+
+    #[inline]
+    pub fn set_cursor_visible(&self, _visible: bool) {}
+
+    #[inline]
+    pub fn drag_window(&self) -> Result<(), ExternalError> {
+        Err(ExternalError::NotSupported(NotSupportedError::new()))
+    }
+
+    #[inline]
+    pub fn set_cursor_hittest(&self, _hittest: bool) -> Result<(), ExternalError> {
+        Err(ExternalError::NotSupported(NotSupportedError::new()))
+    }
+
+    #[inline]
+    pub fn scale_factor(&self) -> f64 {
+        1.0
+    }
+
+    #[inline]
+    pub fn set_cursor_position(&self, _position: Position) -> Result<(), ExternalError> {
+        Err(ExternalError::NotSupported(NotSupportedError::new()))
+    }
+
+    #[inline]
+    pub fn set_maximized(&self, _maximized: bool) {}
+
+    #[inline]
+    pub fn is_maximized(&self) -> bool {
+        true
+    }
+
+    #[inline]
+    pub fn set_minimized(&self, _minimized: bool) {}
+
+    #[inline]
+    pub fn fullscreen(&self) -> Option<Fullscreen> {
+        Some(Fullscreen::Exclusive(crate::monitor::VideoMode {
+            video_mode: crate::platform_impl::VideoMode::Drm(super::VideoMode(
+                self.0,
+                self.1.clone(),
+            )),
+        }))
+    }
+
+    #[inline]
+    pub fn set_fullscreen(&self, _monitor: Option<Fullscreen>) {}
```

Comment:
And this call is actually possible, since it's the one that changes video modes with exclusive fullscreen.

Reply:
Fixed